locale/en/messages.json

```diff
@@ -765,10 +766,11 @@
   "pld.requiescat.suggestions.missed-confiteor.content": "Be sure to use <0/>, <1/>, <2/>, and <3/> in each <4/> window for optimal damage.",
   "pld.requiescat.table.note": "Each of your <0/> windows should contain 4 spells , consisting of <1/>, <2/>, <3/>, and <4/> for each each stack <5/>.",
   "pld.requiescat.title": "Requiescat Usage",
+  "pld.royal-authority.checklist.description": "<0/> generates <1/> to use on <2/> and follow up actions <3/> and <4/>. This is effectively the same as getting {0} uses of <5/> or stronger actions.",
+  "pld.royal-authority.checklist.name": "Use Follow Up Skills of Royal Authority",
   "pld.royal-authority.title": "Royal Authority",
-  "pld.sword-oath.checklist.description": "<0/> generates {swordOathStacks} stacks of <1/> to use on <2/>. This is effectively the same as getting {swordOathStacks} uses of <3/>.",
-  "pld.sword-oath.checklist.name": "Use Sword Oath stacks generated by Royal Authority",
-  "pld.sword-oath.checklist.requirement.name": "Uses of <0/> out of possible uses",
+  "pld.sepulchre.checklist.requirement.name": "Uses of <0/> out of possible uses",
+  "pld.supplication.checklist.requirement.name": "Uses of <0/> out of possible uses",
   "pld.timeouts.confiteor.content": "Try to consume <0/> before it expires as the <1/> combo is your strongest combo.",
```

Comment:
I think the locale need cleaned up a bit. Just replacing sword oath with royal authority should work imo. I don't think there's a point for the atonement, supplication, and sepulchre checklists. I do like how you worded the FoF suggestions!

Reply:
Locale files should not be edited directly. The naming of the entries is controlled by the "id" field of the Trans tags used within module files.

Reply:
I believe the issue is that I had the same text 3 times but 3 different keys. For an example see the list of the screenshot of the Royal Authority Analysis in PR Description. I use the same key for all three now, which also cleans up the messages a bit.

Though I'm not entirely sure if it causes issues how I do it now. Is there a better way to "define" a Translation at one point and then "use" it at another?

Reply:
@AzariahAeroborn We kind of have to in order to adjust the English portion to be correct, right? I get that with the id fields though.

Reply:
No, English translations should **only** be updated via `yarn extract`, after putting the desired copy in the Trans tag in the module that has the copy.

Reply:
> I believe the issue is that I had the same text 3 times but 3 different keys. For an example see the list of the screenshot of the Royal Authority Analysis in PR Description. I use the same key for all three now, which also cleans up the messages a bit.
> 
> Though I'm not entirely sure if it causes issues how I do it now. Is there a better way to "define" a Translation at one point and then "use" it at another?

Judging by the documentation here: https://lingui.dev/tutorials/explicit-vs-generated-ids - as long as you have the exact same copy in the Trans tag, you could re-use the same id.  However, if the copy varies or you're using other properties in the tag to give extra context to the translators, then repeating IDs won't work because lingui is going to grab the first string it sees with a given ID and ignore the rest of them.

Reply:
I did run `yarn extract`. I remember that we had a similar discussion when I marked atonement non-combo breaking in 6.4. I've not adjusted any texts manually.